modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/application/DefaultMuleApplication.java

```diff
@@ -299,6 +299,7 @@ private void doInit(boolean lazy, boolean disableXmlValidations, boolean addTool
       }
       artifactContext = artifactBuilder.build();
       setMuleContext(artifactContext.getMuleContext(), artifactContext.getRegistry());
+      LOGGER.debug(artifactContext.getMuleContext().getStatistics().getFlowSummaryStatistics().toString());
```

Comment:
maybe this has to be INFO?

Reply:
GUS ticket has defined the log level as DEBUG